app/views/api/v1/positions/create.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.position do
```

Comment:
`json.extract! @position, :id, :name, :salary, :vacation_days`
It's more readable and easier to write. Apply it for all appropriate views.